modules/instrument_manager/php/instrument_manager.class.inc

```diff
@@ -168,25 +155,40 @@ class Instrument_Manager extends \NDB_Menu_Filter
              * installation is disabled (ie. has no quatUser), as the above
              * error will return before this one.
             */
-            return (new \Loris\Http\Response())
+            return (new \LORIS\Http\Response())
                 ->withStatus(409, 'Conflict')
                 ->withBody(
                     new \LORIS\Http\StringStream(
                         json_encode(
-                            array('error' => self::FILE_ALREADY_EXISTS)
+                            array('error' => self::ALREADY_INSTALLED)
```

Comment:
i don't think you should remove the `FILE_ALREADY_EXISTS` case here. it's useful to know when the instrument has been uploaded but not installed

Reply:
I changed the code so that the FILE ALREADY EXISTS case now gets handled further down in the file. ALREADY INSTALLED is more appropriate here

Reply:
then i think the comments above here need to be moved down to apply to the case for FILE_ALREADY_EXISTS, not for ALREADY_INSTALLED